package.json

```diff
@@ -49,6 +49,7 @@
     "eslint-plugin-markdown": "^3.0.1",
     "eslint-plugin-prettier": "^5.1.3",
     "eslint-plugin-react": "^7.34.2",
+    "eslint-plugin-react-hooks": "^4.6.2",
```

Comment:
Had to move this plugin to the root, as it is used by the linting configuration of the project. It was previously included incidentally as a dependency of the ESLint plugin.